values.yaml

```diff
@@ -1040,7 +1040,7 @@ ui:
     provider: "prometheus"
 
     # baseURL is the URL of the prometheus server, usually the service URL.
```

Comment:
L1038 also needs ui.enabled in backticks

Reply:
good catch!!